test/custom-parser.test.js

```diff
@@ -312,7 +323,7 @@ test('the content type should be a string', t => {
     fastify.addContentTypeParser(null, () => {})
     t.fail()
   } catch (err) {
-    t.is(err.message, 'The content type should be a string')
+    t.is(err.message, new FST_ERR_CTP_INVALID_TYPE().message)
```

Comment:
I would prefer for the message to be explicit here.

Reply:
What's the benefit? I think the error name is already very explicit. In that way we can avoid modifying all tests when we change something in the future.

Reply:
Sure, but I would love to avoid testing all errors individually.